### ISSUE TITLE
Need goto type definition/ peek type definition

### ISSUE THREAD

**Issue by cloud-yu** [microsoft/vscode-cpptools#7999]
a common usage:

```
typedef struct test
{
    int first;
    int second;
} TEST;

void func()
{
    TEST t1;
}
```
goto/peek definition on `t1` will show `TEST t1;`,  need a goto/peek type definition show the `typedef` part

**Comment by sean-mcmanus** [comment]
Is doing Go to Definition on TEST sufficient?

**Comment by cloud-yu** [comment]
doing go to definition on TEST works perfect.
in a long function content, usually we use the variable `t1`, there would  two steps got type difinition. 
goto definition on t1, editor jump to `TEST t1` , then goto/peek definition on `TEST`.

this feature may be trivial, but could be useful and efficient? Right?

**Comment by sean-mcmanus** [comment]
My reference to "trivial" was in regards to the implementation and not the feature itself -- it should be available in 1.7.0.

**Comment by sean-mcmanus** [comment]
Go to Type Definition is available with https://github.com/microsoft/vscode-cpptools/releases/tag/1.7.0-insiders2
